src/modules/keyboard/Config.h

```diff
@@ -58,6 +59,9 @@ class Config : public QObject
      * (dvorak).
      */
     KeyboardVariantsModel* keyboardVariants() const;
+    /* A group describes a toggle groups of change layouts
+     */
+    KeyboardGroupsModel* keyboardGroups() const;
```

Comment:
Let's call it a *GroupSwitcher* then, so `KeyboardGroupSwitcherModel` and `keyboardGroupSwitchers()`